.github/workflows/nightly-doc-build.yml

```
@@ -53,15 +57,23 @@ jobs:
         env:
           ANSYSLMD_LICENSE_FILE: ${{ format('1055@{0}', secrets.LICENSE_SERVER) }}
           PYFLUENT_START_INSTANCE: 0
-          DOCS_CNAME: dev.fluentvisualization.docs.pyansys.com
           FLUENT_IMAGE_TAG: ${{ matrix.image-tag }}
 
+      - name: Upload HTML Documentation
+        uses: actions/upload-artifact@v3
+        with:
+          name: HTML-Documentation-tag-${{ env.DOC_DEPLOYMENT_IMAGE_TAG }}
+          path: doc/_build/html
+
+          retention-days: 7
+
       - name: Deploy
-        if: matrix.image-tag == 'v23.1.0'
-        uses: JamesIves/github-pages-deploy-action@v4.4.1
+        if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
+        uses: pyansys/actions/doc-deploy-stable@v2
```

You can use the latest actions:

```suggestion
        uses: pyansys/actions/doc-deploy-stable@v3
```